examples/small-study/src/background.js

```diff
@@ -0,0 +1,76 @@
+// Allow outside override for debug and testing using prefs
+const STUDYPREFS = {
```

Comment:
We can have browser.study.getPref and browser.study.setPref automatically prefix the arguments with `shield.${addonWidgetId}.` and remove it from the boilerplate

Reply:
Good points.  For now, keeping it explicit for these reasons:

1.  Moved prefs stuff to a different API.
2.  The autoprefixing has caused a lot of trouble in the past, becuaes things change names, the control prefs move, and all kinds of unexpected stuff.